.circleci/config.yml

```diff
@@ -63,13 +63,12 @@ jobs: # a collection of steps
           path: backend/reports/coverage/lcov-report
           destination: backend-coverage
       - run:
-          name: copy frontend coverage
-          command: cp ./frontend/coverage/lcov.info ./backend/reports/coverage/lcov_frontend.info  
-      - run:
-          name: merge frontend and backend coverage
-          command: ./backend/node_modules/.bin/lcov-result-merger './backend/reports/coverage/lco*.info' './backend/reports/coverage/lcov_merged.info'
+          command: cat ./frontend/coverage/lcov.info | ./backend/node_modules/.bin/coveralls
```

Comment:
'name:' ?

Reply:
I am playing with it, it is not ready for review yet ;)